README.md

```diff
@@ -16,62 +16,66 @@ To use:
-* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_gazebo6-install_bottle-homebrew-amd64)](https://build.osrfoundation.org/view/ign-fortress/job/ignition_gazebo6-install_bottle-homebrew-amd64/)         [ignition_gazebo6](https://build.osrfoundation.org/view/ign-fortress/job/ignition_gazebo6-install_bottle-homebrew-amd64/)
-* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_launch5-install_bottle-homebrew-amd64)](https://build.osrfoundation.org/view/ign-fortress/job/ignition_launch5-install_bottle-homebrew-amd64/)         [ignition_launch5](https://build.osrfoundation.org/view/ign-fortress/job/ignition_launch5-install_bottle-homebrew-amd64/)
-* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_fortress-install_bottle-homebrew-amd64)](https://build.osrfoundation.org/view/ign-fortress/job/ignition_fortress-install_bottle-homebrew-amd64/)       [ignition_fortress](https://build.osrfoundation.org/view/ign-fortress/job/ignition_fortress-install_bottle-homebrew-amd64/)
+* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_physics5-install_bottle-homebrew-amd64)](https://build.osrfoundation.org/view/ign-fortress/job/ignition_physics5-install_bottle-homebrew-amd64/)       [gz-physics5](https://build.osrfoundation.org/view/ign-fortress/job/ignition_physics5-install_bottle-homebrew-amd64/)
+* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_rendering6-install_bottle-homebrew-amd64)](https://build.osrfoundation.org/view/ign-fortress/job/ignition_rendering6-install_bottle-homebrew-amd64/)   [gz-rendering6](https://build.osrfoundation.org/view/ign-fortress/job/ignition_rendering6-install_bottle-homebrew-amd64/)
+* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_gui6-install_bottle-homebrew-amd64)](https://build.osrfoundation.org/view/ign-fortress/job/ignition_gui6-install_bottle-homebrew-amd64/)               [gz-gui6](https://build.osrfoundation.org/view/ign-fortress/job/ignition_gui6-install_bottle-homebrew-amd64/)
+* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_sensors6-install_bottle-homebrew-amd64)](https://build.osrfoundation.org/view/ign-fortress/job/ignition_sensors6-install_bottle-homebrew-amd64/)       [gz-sensors6](https://build.osrfoundation.org/view/ign-fortress/job/ignition_sensors6-install_bottle-homebrew-amd64/)
+* [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_gazebo6-install_bottle-homebrew-amd64)](https://build.osrfoundation.org/view/ign-fortress/job/ignition_gazebo6-install_bottle-homebrew-amd64/)         [gz-gazebo6](https://build.osrfoundation.org/view/ign-fortress/job/ignition_gazebo6-install_bottle-homebrew-amd64/)
```

Comment:
`gz-sim6` instead of `gz-gazebo6`?

Reply:
thanks; fixed in https://github.com/osrf/homebrew-simulation/pull/2267/commits/d2f305f96ea2bb2c0de52b19f58e4a1b55862096